app/scripts/directives/processTemplateDialog.js

```diff
@@ -184,7 +185,7 @@
     };
 
     ctrl.templateProjectChange = function () {
-      ctrl.templateProjectName = _.get(ctrl.templateProject, 'metadata.name');
+      ctrl.templateProjectName = _.get(ctrl.selectedProject, 'metadata.name');
```

Comment:
I kind of like the original `templateProject` name better. `selectedProject` sounds like the project you are instantiating the template into.